mava/systems/ippo/networks.py

```diff
@@ -256,24 +256,29 @@ def policy_fn(inputs: jnp.ndarray) -> networks_lib.FeedForwardNetwork:
             FeedForwardNetwork class
         """
         # Add the observation network and an MLP network.
-        policy_network = [
-            MLP_NORM(
-                policy_layer_sizes,
-                activation=activation_function,
-                w_init=w_init_fn(orthogonal_initialisation, jnp.sqrt(2)),
-                activate_final=True,
-                layer_norm=layer_norm,
-            ),
-        ]
+        policy_layers = []
+        if observation_network:
+            policy_layers.extend([observation_network])
+        policy_layers.extend(
```

Comment:
Minor comment, but we can probably just append for both the above lines.

Reply:
This is the only question I have as well.

Reply:
Valid point. I changed it.

Reply:
I switched everything back to `extend` for consistency (better than having 3 appends and 1 extends).

Reply:
I feel appends are cleaner for the first 3. But up to you :slightly_smiling_face: